### DISCUSSION TITLE
Adding a new module

### DISCUSSION THREAD

**Discussion by Pedro-Floriano** [mboquien/cigale#63] [Q&A]
Hello everyone, I would like to know if it is possible, and if so, how could I be adding a new module into CIGALE. The module is the X-Shooter Spectral Library (XSL), it would really help on my research.

Reference: https://www.aanda.org/articles/aa/full_html/2022/04/aa42388-21/aa42388-21.html

**Comment by mboquien** [comment]
Hello. In its current state, the library should be extended in the UV (it stops at 350 nm, so the UV and the Lyman continuum are missing) and in the IR beyond 2.5 μm. Then the stellar spectra need to be combined based on the metallicity and the IMF. If such libraries were to be made available, I would be happy to integrate them in CIGALE.
